index.html

```diff
@@ -0,0 +1,198 @@
+                                    <a class="nav-link" href="#">Program</a>
+                                </li>
+                                <li class="nav-item">
+                                    <a class="nav-link" href="#">Join</a>
+                                </li>
+                                <li class="nav-item">
+                                    <a class="nav-link" href="#">Sponsor</a>
+                                </li>
+                                <li class="nav-item">
+                                    <a class="nav-link" href="#">News</a>
+                                </li>
+                                <li class="nav-item ms-5">
+                                    <a class="navbar-brand d-none d-lg-block campaign" href="#">Star</a>
+                                </li>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+            </nav>
+
+           <div class="container">
+            <div class="row">
+                <div class="col head_content">
+                    <div class="mt-5 active rounded ">
+                        <h6>"Hello! Sharing world"</h6>
+                        <div id="demo-text" class=" fw-bolder px-1 ">STAR EDUCATIONAL<br>SOCIETY 2022</div>
+                        <P class="tx-custom-black white-border fs-small px-2 py-2 mt-4 mb-4">Star Educational Society is
+                            a not-for-profit educational institute focused on providing access to qu</P>
+                        <h4 class="tx-custom-black fw-bolder ">2015.10.15(THU) ~ 16(FRI)</h4>
+                        <P class="tx-custom-black fs-small">@ National Museum of Korea, Art Center Nabi and more</P>
+                    </div>
+                </div>
+            </div>
+           </div>
+
+        </div>
+    </div>
+    <section class="section bg-custom-black container-fluid">
+        <div class="container bg-transparent">
+            <div class="row text-white px-1 py-3 text-center">
+                <h5>Main Program</h5>
+                <hr class="hr">
+            </div>
+
+            <div class="row program_section">
+                <div class="program">
+                    <div class="program_item">
+                        <div class="img_cover">
+                            <img src="assets/images/discoursew.png" width="40px" alt="">
+                        </div>
+                        <div class="title">
+                            <h6 class="tx-custom-color">Lecture</h6>
+                        </div>
+                        <div class="desc">
+                            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. </p>
+                        </div>
+                    </div>
+
+                    <div class="program_item">
+                        <div class="img_cover"><img src="assets/images/galery.png" width="40px" alt=""></div>
```

Comment:
- [Optional] Dear @Elhambasir, Please add text to your `alt` attributes of an image in all your `img` tag. This has an advantage when it comes to semantics. Screen readers and SEO needs such information to better understand your page.